generators/app/templates/static/scss/utilities/_utilities.content-editor.scss

```diff
@@ -0,0 +1,366 @@
+            transition: getTransition(default, ease-in-quad);
+
+            @include hover {
+            }
+        }
+
+        &.is-style-regular {
+        }
+    }
+
+    .wp-block-quote {
+        margin-top: var(--s-60);
+        margin-bottom: var(--s-60);
+        padding-left: var(--s-15);
+        border-left: 3px solid $black;
+
+        &:first-child {
+            margin-top: 0;
+        }
+
+        &:last-child {
+            margin-bottom: 0;
+        }
+
+        p {
+            @include a2;
+        }
+
+        cite {
+            @include b1;
+
+            margin-top: var(--s-15);
+            display: block;
+
+            > u {
+                text-decoration: underline;
+            }
+
+            > a {
+                text-decoration: underline;
+                transition: getTransition(default, ease-in-quad);
+
+                @include hover {
+                }
+            }
+        }
+
+        &.is-style-large {
+        }
+    }
+
+    .wp-block-embed {
+        margin-top: var(--s-60);
+        margin-bottom: var(--s-60);
+
+        &:first-child {
+            margin-top: 0;
+        }
+
+        &:last-child {
+            margin-bottom: 0;
+        }
+
+        &.is-type-video {
+            .wp-block-embed__wrapper {
+                position: relative;
+                padding-bottom: 56.25%;
+                height: 0;
+                overflow: hidden;
+                max-width: 100%;
+
+                iframe,
+                object,
+                embed {
+                    position: absolute;
+                    top: 0;
+                    left: 0;
+                    width: 100%;
+                    height: 100%;
+                }
+            }
+        }
+    }
+
+    .wp-block-image {
+        margin-top: var(--s-60);
+        margin-bottom: var(--s-60);
+
+        &:first-child {
+            margin-top: 0;
+        }
+
+        &:last-child {
+            margin-bottom: 0;
+        }
+
+        &.size-full {
+        }
+
+        &.size-large {
+        }
+
+        &.size-medium {
+            img {
+                width: auto;
+            }
+        }
+
+        &.size-thumbnail {
+            img {
+                width: auto;
+            }
+        }
+
+        figcaption {
+            @include b1;
+
+            margin-top: var(--s-15);
+
+            > u {
+                text-decoration: underline;
+            }
+
+            > a {
+                text-decoration: underline;
+                transition: getTransition(default, ease-in-quad);
+
+                @include hover {
+                }
+            }
+        }
+    }
+
+    .c-gutenberg-block {
+        margin-top: var(--s-60);
+        margin-bottom: var(--s-60);
+        position: relative;
+        z-index: getZindex(default);
+
+        &:first-child {
+            margin-top: 0;
+        }
+
+        &:last-child {
+            margin-bottom: 0;
+        }
+    }
+}
```

Comment:
dodo bi na kraj ovo

    > *:first-child {
        margin-top: 0;
    }

    > *:last-child {
        margin-bottom: 0;
    }
    
    znaci prvi item u gutnbergu mi ne treba imat margin top, a zadnji item margin buttom, nebitno koji je item

Reply:
Javim ti onda jos za ovaj gutenberg, kuzim kaj si napravio, radim malo drugacije ali mi je oke ovo, i nebi da bude blocker za merge, moze ic ovo tvoje

Reply:
ajmo ovaj cijeli file komentirati na nekom sastanku pa da imamo usporedno tvoj i moj fajl, kak smo se dogovorili na meetu ako ti je to ok?